libs/wingc/src/jsify.rs

```diff
@@ -411,6 +455,9 @@ impl<'a> JSifier<'a> {
 								ExprKind::Reference(Reference::InstanceMember { object, .. }) => {
 									self.jsify_expression(&object, ctx)
 								}
+                ExprKind::Reference(Reference::TypeMember { .. }) => {
+                  expr_string.clone().split(".").next().unwrap().to_string()
```

Comment:
Have a feeling this might break with namespaces: `Namespace.Namesapce.Type.property`

Reply:
I added an `unwrap_or()`